src/producer.cpp

```diff
@@ -94,6 +109,20 @@ uint32_t IceflowProducer::getNextPartitionNumber() {
   return m_randomNumberGenerator() % m_numberOfPartitions;
 }
 
+// TODO: Determine where to use this method.
```

Comment:
We'd probably have to integrate with NDN-SVS in order to manage the queue for a given edge (both to delete already processed elements and to detect processing congestions based on the queue size).

Reply:
Yeah, that makes sense. Should we create an issue to keep track of that or do you think keeping the TODO wold suffice here?

Reply:
Keeping the TODO will probably suffice.